docs/topics/whatsnew-eap.md

```diff
@@ -239,9 +239,17 @@ This option enables a preview of the new import mode. Besides the support for co
 import experience in multiplatform projects, as we included major bug fixes and improvements to make the import more
 stable.
 
-Please keep in mind that it's still a preview version that needs further stabilization, and you might encounter some
-issues with import along the way. We encourage you to try it out and submit all reports
-in [YouTrack](https://kotl.in/issue) to help us make it default in Kotlin 1.9.0.
+### Known issues
+
+Please keep in mind that it's still a preview version that needs further stabilization, and you might encounter some issues
+with import along the way. Here are some known issues we're planning to fix before the final release of Kotlin 1.8.20:
+
+* There's no Kotlin 1.8.20 plugin available for IntelliJ IDEA 2023.1 EAP yet. Despite that, you can still set the Kotlin
+  Gradle plugin version to 1.8.20-Beta and try out composite builds in this IDE.
+* If your projects have builds with specified `rootProject.name`, composite builds may fail to resolve the Kotlin metadata.
```

Comment:
> If your projects have builds

-> "If your project include builds...". Here, verb "include" refers to special Gradle operation connected with "included builds" feature, so it's not possible to replace it with synonym-verbs with meaning "contain"